openshift/rabbitmq.yaml

```diff
@@ -237,3 +240,18 @@ parameters:
   name: RABBITMQ_ERLANG_COOKIE
   from: '[a-zA-Z0-9]{16}'
   generate: expression
+- description: Maximum amount of memory the app container can use.
+  displayName: Memory Limit
+  name: MEMORY_LIMIT
+  required: true
+  value: 1Gi
+- description: Maximum amount of CPU the app container can use.
+  displayName: CPU Limit
+  name: CPU_LIMIT
+  required: true
+  value: '1'
+- displayName: Volume Capacity
+  description: Volume space available for shared files, e.g. 512Mi, 2Gi
+  name: VOLUME_CAPACITY
+  required: true
+  value: 1Gi
```

Comment:
no idea if lack of new line is a problem or not but just wanted to mention it in case it is.